README.md

```diff
@@ -89,24 +91,98 @@ const ckanPackage = frictionlessCkanMapper.packageFrictionlessToCkan(
 )
 
 console.log(ckanPackage)
-// {
```

Comment:
generally i would terminology of dataset consistently rather than package going forward including here.

So terminology is `Dataset` and `Resource`